### PR TITLE
feat: update dummy paymaster and data

### PR DESCRIPTION
**Pull request by avasisht23**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
This PR adds a new function `getAlchemyPaymasterAddress` to dynamically retrieve paymaster addresses based on the chain and updates gas management middleware to use this function.

### Detailed summary
- Added `getAlchemyPaymasterAddress` function to dynamically fetch paymaster addresses based on the chain.
- Updated gas management middleware to use `getAlchemyPaymasterAddress` function for determining paymaster addresses.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: packages/alchemy/src/middleware/gasManager.ts [hunk 1]
```diff
@@ -40,14 +55,32 @@ export interface AlchemyGasEstimationOptions {
...
-      case 1:
-      case 10:
-      case 137:
-      case 42161:
-        return "0x4Fd9098af9ddcB41DA48A1d78F91F1398965addcfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0000000000000000000000000000000007aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa1c";
+    const paymasterAddress = "0x4Fd9098af9ddcB41DA48A1d78F91F1398965addc";
+    const olderPaymasterAddress = "0xc03aac639bb21233e0139381970328db8bceeb67";
+    const arbSepoliaPaymasterAddress =
+      "0x0804Afe6EEFb73ce7F93CD0d5e7079a5a8068592";
+
```
Comment (noam-alchemy) [inline packages/alchemy/src/middleware/gasManager.ts:62]: should we structure these as global constants in `alchemy/`?

File: packages/alchemy/src/middleware/gasManager.ts [hunk 2]
```diff
@@ -40,14 +55,32 @@ export interface AlchemyGasEstimationOptions {
...
+    const dummyData =
```
Comment (noam-alchemy) [inline packages/alchemy/src/middleware/gasManager.ts:63]: same - global constant? what accounts does this work and not work for?

File: packages/alchemy/src/middleware/gasManager.ts (hunk 1 above)
Comment (moldy530) [reply packages/alchemy/src/middleware/gasManager.ts:62]: Yeah +1 to this can we add this to a get paymaster address method or something?

**Review by moldy530 — DISMISSED** [review]
Approving to unblock, let's follow up with the utils

**Review by rthomare — APPROVED** [review]
Let's follow up here after fixing this for the customer!

File: packages/alchemy/src/middleware/gasManager.ts (hunk 2 above)
Comment (avasisht23) [reply packages/alchemy/src/middleware/gasManager.ts:46]: let's keep this in the dummy constant for now since it's not used elsewhere

**Pull request merged**